daml-lf/archive/src/test/scala/com/digitalasset/daml/lf/archive/DecodeV1Spec.scala

```diff
@@ -999,6 +1000,60 @@ class DecodeV1Spec
       }
     }
 
+    s"Reject interface definitions in modules iff version < ${LV.Features.interfaces}" in {
+      val i = DamlLf1.DefInterface.newBuilder()
+      i.setTyconInternedDname(1)
+      i.setParamInternedStr(0)
+      i.setPrecond(unitExpr)
+
+      val m = DamlLf1.Module.newBuilder()
+      m.setNameInternedDname(0)
+      m.addInterfaces(i.build())
+      m.setFlags(
+        DamlLf1.FeatureFlags
+          .newBuilder()
+          .setForbidPartyLiterals(true)
+          .setDontDivulgeContractIdsInCreateArguments(true)
+          .setDontDiscloseNonConsumingChoicesToObservers(true)
+      )
+
+      val ifaceTemplateScala =
+        GenModule(
+          Ref.DottedName.assertFromString("Mod"),
+          Map(),
```

Comment:
(just a cosmetic advice) 

I prefer using the `empty` function as it avoid in general unnecessary allocation.  

```suggestion
          Map.empty,
```